py-doc-to-xml/docx_to_xml/types.py

```diff
@@ -25,6 +25,28 @@ class SemanticDomain:
     description: str
     questions: List[str]
 
-    def is_valid(self):
+    def is_valid(self) -> bool:
         return is_semantic_domain_number(self.number)
-    
\ No newline at end of file
+
+    # # Define the comparison operators so that Python will sort for us
```

Comment:
Remove extra `# ` at start of these comments?

Reply:
Done.